### DISCUSSION TITLE
Is there a rule to set the order of two imports (type and not) from the same file?

### DISCUSSION THREAD

**Discussion by silversonicaxel** [eslint/eslint#18141] [Q&A]
Is there a rule to set the order of two imports from the same file, when one is a type import and the second is for objects?

```
import type { CarProps } from 'car';
import { Car, CarKeys } from 'car';
```

I would like to have the example shown before as the correct one, while the following one should be labelled for order error

```
import { Car, CarKeys } from 'car';
import type { CarProps } from 'car';
```

Is there a rule that I missed?

**Comment by nzakas** [comment]
Not in the core, no. We don't have TypeScript-specific rules in the core. You'd need to check the typescript-eslint plugin to see if they have anything.
